Sources/PackageDescription/Target.swift

```diff
@@ -134,6 +140,10 @@ public final class Target {
     /// The type of the target.
     public let type: TargetType
 
+    /// The group this target belongs to, where access to the target's group-specific
+    /// APIs is not allowed from outside.
+    public let group: TargetGroup
```

Comment:
If possible, this should have availability decls.

Reply:
same as above